portal/src/pages/my/privacy.vue

```diff
@@ -125,20 +126,42 @@ export default {
       if (isNil(this.user) || isNil(this.user.permissions)) {
         return []
       }
-      return (
+      let permissions =
```

Comment:
Ik zou hier een `const` van maken en dan returnen in die `if` loop. Maar dat is denk ik meer een kwestie van smaak 😛 .